uportal-war/src/main/java/org/jasig/portal/portlets/portletadmin/PortletDefinitionForm.java

```diff
@@ -555,6 +561,34 @@ public Date getExpirationDateTime() {
 		return cal.getTime();
 	}
 
+
+	/**
+	 * Get the expiration date as a string.  This is
+	 * @return the expiration date as a string
+	 */
+	public String getExpirationDateString() {
+		if (expirationDate == null) {
+			return null;
+		}
+
+		return dateFormat.format(expirationDate);
+	}
+
+
+	/**
+	 * Set the expiration date as a string.
+	 * @param date the date string
+	 * @throws ParseException if the string cannot be parsed
+	 */
+	public void setExpirationDateString(String date) throws ParseException {
```

Comment:
Yes.   I can't recall the exact details off the top of my head, but it had to do with the date webflow binding.   If I used the default bindings, it worked, but then I couldn't use the date converter that is configured for uPortals webflow.   If I tried to specify the bindings it would not correctly handle the parameter and preferences maps.